tsfile/src/main/java/org/apache/iotdb/tsfile/read/common/block/TsBlock.java

```diff
@@ -213,6 +214,14 @@ public Column[] getColumns(int[] columnIndexes) {
     return columns;
   }
 
+  public TSDataType[] getValueDataTypes() {
+    TSDataType[] dataTypes = new TSDataType[valueColumns.length];
+    for (int i = 0; i < valueColumns.length; i++) {
+      dataTypes[i] = valueColumns[i].getDataType();
+    }
+    return dataTypes;
+  }
```

Comment:
Better not to try to get TsDataType info from TsBlock, because we may make TsBlock typeless which means Float and Int share the same Int32Column.